src/lib/rust.ts

```diff
@@ -285,19 +286,19 @@ const tryDownloadingBinary = async ({
 
   const url = `https://github.com/fluencelabs/marine/releases/download/${name}-v${version}/${name}-${platformToUse}-x86_64`;
 
-  try {
-    await downloadFile(binaryPath, url);
-  } catch {
-    return `Failed to download ${name}@${version} from ${url}`;
-  }
+  startSpinner(
+    `Downloading ${name}@${version} binary to ${replaceHomeDir(
+      dependencyDirPath
+    )}`
+  );
```

Comment:
```suggestion
  startSpinner(`Downloading ${name}@${version} binary to ${replaceHomeDir(dependencyDirPath)}`);
```

to me it's much easier to read, but if for you it's not, I'm ok with either

Reply:
the formatiing is done with prettier, no easy ways to control it and it's a good thing - less to argue about on code reviews

Reply:
yep, makes sense